src/transaction/types.ts

```diff
@@ -15,6 +15,11 @@ import type {
 
 export type Call = { to: Hex; data?: Hex; value?: bigint };
 
+type TransactionButtonOverride = {
+  text?: string;
```

Comment:
we are stuck with the text prop name, but any reason not to allow a ReactNode here?  this would allow someone to use their own custom loading icon for the pending state, etc.

Reply:
yeah i agree i'll update type but keep `text` name for consistency since i don't think it is worth causing a breaking change